src/openforms/formio/registry.py

```diff
@@ -27,22 +27,21 @@
 if TYPE_CHECKING:
     from openforms.submissions.models import Submission
 
+T = TypeVar("T", bound=Component, contravariant=True)
```

Comment:
I like having the actual bound type specified in the name, makes it clearer in my opinion: `ComponentT`.

But this is just personal preference